### PR TITLE
fix: outdated shortcuts

### PR DESCRIPTION
**Pull request by danil-pavlov**
This PR fixes [KT-69612](https://youtrack.jetbrains.com/issue/KT-69612), outdated shortcuts for iOS targets

### CODE REVIEW

File: docs/topics/native/native-cocoapods-dsl-reference.md [hunk 1]
```diff
@@ -52,18 +52,21 @@ You can use the following blocks, functions, and properties inside it:
...
+|                     |              |                      | `watchosDeviceArm64`    |
 
 For each target, use the `deploymentTarget` property to specify the minimum target version for the Pod library.
 
 When applied, CocoaPods adds both `debug` and `release` frameworks as output binaries for all of the targets.
 
 ```kotlin
 kotlin {
-    ios()
+    iosX64()
```
Comment (daugeldauge) [inline docs/topics/native/native-cocoapods-dsl-reference.md:69]: Should we use maybe a `iosArm64` as a default? `iosX64` is slowly going away I think, as x86 macs are not available anymore

**Review by daugeldauge — APPROVED** [review]

File: docs/topics/native/native-cocoapods-dsl-reference.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/native/native-cocoapods-dsl-reference.md]: That makes sense, thank you! Updated throughout the docs

File: docs/topics/multiplatform/multiplatform-ios-dependencies.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ import DateTools.*
 
     ```groovy
     kotlin {
-        iosX64 {
+         iosArm64 {
```
Comment (koshachy) [inline docs/topics/multiplatform/multiplatform-ios-dependencies.md:231]: Looks like there is an extra whitespace character

**Review by koshachy — APPROVED** [review]
LGTM.
Pay attention to a small comment, please.

**Pull request merged**